RealmModelGenerator/AttributesRelationshipsViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AttributesRelationshipsViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/31/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+@objc protocol AttributesRelationViewControllerDelegate {
+    optional func attributesRelationshipsViewController(attributesRelationshipsViewController:AttributesRelationshipsViewController, selectionChange index:Int)
```

Comment:
selection change should not be an index but the actual attribute that was selected